packages/manager/modules/pci/src/projects/project/instances/instance/instance.routing.js

```diff
@@ -18,6 +20,18 @@ export default /* @ngInject */ ($stateProvider) => {
       ) => PciProjectsProjectInstanceService
         .getInstancePrice(projectId, instance),
 
+      reverseDnsLink: /* @ngInject */ coreConfig => DEDICATED_IPS_URL[coreConfig.getRegion()],
+
+      ipMitigationLink: /* @ngInject */ (
+        coreConfig,
+        instance,
+      ) => MITIGATION_URL(instance.publicIpV4[0].ip, coreConfig.getRegion()),
```

Comment:
can we use some lodash "get" for publicIpV4[0].ip? to avoid any null reference error